uber/config.py

```diff
@@ -486,6 +522,7 @@ def _unrepr(d):
         c.PRICE_LIMITS[int(_opt)] = _val
     else:
         c.PRICE_BUMPS[date] = _val
+c.ORDERED_PRICE_LIMITS = sorted([val for key, val in c.PRICE_LIMITS.items()])
```

Comment:
That's what I started with! Unfortunately, it's actually really difficult to go to the 'next' item in an OrderedDict. StackOverflow had some suggestions, but they looked real sketch: https://stackoverflow.com/questions/12328184/how-to-get-the-next-item-in-an-ordereddict